index.html

```diff
@@ -1,89 +1,84 @@
-  <link rel="icon" type="image/png" sizes="32x32" href="./media/favicons/favicon-32x32.png">
-  <link rel="icon" type="image/png" sizes="16x16" href="./media/favicons/favicon-16x16.png">
-  <link rel="manifest" href="./media/favicons/site.webmanifest">
-  <!-- Klaro scripts – Cookie consent manager -->
-  <!-- <script defer type="text/javascript" src="https://substrate.dev/js/config.js"></script> -->
-  <script defer type="text/javascript" src="./resource/js/config.js"></script>
-  <!-- <script defer type="text/javascript" src="https://substrate.dev/js/klaro.min.js"></script> -->
-  <script defer type="text/javascript" src="./resource/js/klaro.min.js"></script>
-  <!-- Global site tag (gtag.js) - Google Analytics with Klaro script -->
-  <script async type="opt-in" data-type="application/javascript" data-src="https://www.googletagmanager.com/gtag/js?id=UA-158673459-1" data-name="googleAnalytics"></script>
-  <script type="opt-in" data-type="application/javascript" data-name="googleAnalytics">
-      window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
-      gtag('config', 'UA-158673459-1');
-  </script>
-</head>
-
-<body>
-  <div id="app"> Getting things ready!</div>
-  <script>
-    window.$docsify = {
-      repo: 'https://github.com/shresthagrawal/jslibp2p-chat-tutorial/',
-      loadSidebar: true,
-      coverpage: true,
-      onlyCover: true,
-      themeable: {
-        readyTransition: true, // default
-        responsiveTables: true  // default
-      },
-      auto2top: true,
-      mergeNavbar: true,
-      search: {
-        noData: {
-          '/': 'No results!'
+  <head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta charset="UTF-8">
+    <!-- Theme: Simple Dark -->
+    <!-- <link rel="stylesheet" href="//unpkg.com/docsify/themes/dark.css"> -->
+    <link rel="stylesheet" href="./resource/themes/dark.css">
+    <link rel="stylesheet" href="./custom/custom.css">
+    <link rel="apple-touch-icon" sizes="180x180" href="./media/logo_small.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./media/favicons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./media/favicons/favicon-16x16.png">
+    <link rel="manifest" href="./site.webmanifest">
+    <!-- Klaro scripts – Cookie consent manager -->
+    <!-- <script defer type="text/javascript" src="https://substrate.dev/js/config.js"></script> -->
+    <script defer type="text/javascript" src="./resource/js/config.js"></script>
+    <!-- <script defer type="text/javascript" src="https://substrate.dev/js/klaro.min.js"></script> -->
+    <script defer type="text/javascript" src="./resource/js/klaro.min.js"></script>
+    <!-- Global site tag (gtag.js) - Google Analytics with Klaro script -->
```

Comment:
I think these scripts were modified from the original one, but you can check it again to confirm